app/register/patient/patientForm.tsx

```
@@ -28,20 +28,46 @@ const formSchema = z.object({
-  }, z.number().int().min(0, { message: 'Age must be a positive number' }).max(120, { message: 'Age must be realistic' })),
-  weight: z.preprocess((value) => {
-    if (typeof value === 'string') {
-      const parsed = parseFloat(value);
-      return isNaN(parsed) ? undefined : parsed;
-    }
-    return value;
-  }, z.number().min(0, { message: 'Weight must be a positive number' })),
+  age: z.preprocess(
+    (value) => {
+      if (typeof value === 'string') {
+        const parsed = parseInt(value, 10);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)